### PR TITLE
Player

### PR DESCRIPTION
**Pull request by ThudPoland**
Changes due to implementing player.

### CODE REVIEW

File: player/Player.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+void Player::processLogic(const float deltaTime)
+{
+    if(_isMoving)
+    {
+        sf::Vector2f moveVector;
+        moveVector.x = sin((M_PI/180) * _rotationDegrees);
+        moveVector.y = cos((M_PI/180) * _rotationDegrees);
+        move(moveVector);
+    }
+}
```
Comment (Kicer86) [inline player/Player.cpp]: deltaTime not used + moving by rotationdegrees?

File: world/World.hpp [hunk 2]
```diff
@@ -13,6 +13,7 @@ class World {
     World();
     uint addObject(std::unique_ptr<IObject> object);
     IObject* getObject(uint id);
+    void processLogic(const float deltaTime);
```
Comment (Kicer86) [inline world/World.hpp]: how about chrono?

File: basic/ILogic.hpp [hunk 3]
```diff
@@ -1,12 +0,0 @@
...
-
-#include "IObject.hpp"
-
-class ILogic {
-public:
-    virtual void process(IObject& model) = 0;
-    virtual ~ILogic() = default;
-};
-
-#endif //ILOGIC_HPP
```
Comment (Kicer86) [inline basic/ILogic.hpp]: will std::functions be enough?

File: world/World.hpp (hunk 2 above)
Comment (ThudPoland) [reply world/World.hpp]: I think i'll postpone it on @Xavrax, beacuse he is making changes with chrono

File: player/Player.cpp (hunk 1 above)
Comment (ThudPoland) [reply player/Player.cpp]: Will fix.

File: basic/ILogic.hpp (hunk 3 above)
Comment (ThudPoland) [reply basic/ILogic.hpp]: std::functions is not only posibility

**Pull request merged**